install/operator/pkg/apis/syndesis/v1alpha1/syndesis_types.go

```diff
@@ -112,6 +112,23 @@ type Resources struct {
 	Memory string `json:",inline,omitempty"`
 }
 
+type VolumeAccessMode string
```

Comment:
Creating a new type here only makes sense if you validate the parameter further down. https://github.com/operator-framework/operator-sdk/blob/master/doc/user-guide.md#openapi-validation

Reply:
Added the enum validation to the type. Not made any difference to any other file after re-running sdk-operator k8s.